packages/x-date-pickers/src/locales/index.ts

```diff
@@ -36,4 +36,5 @@ export * from './urPK';
 export * from './viVN';
 export * from './zhCN';
 export * from './zhHK';
+export * from './ar';
```

Comment:
Based on other languages, it is recommended to specify the country of the language, such as arMR or arEG

Reply:
@ET-TOUNANI As arabic is spoken in many countries they have many dialects just like eu. So i have taken that consideration and named it. Kindly suggest!